dask/dataframe/accessor.py

```diff
@@ -147,6 +147,8 @@ def split(self, pat=None, n=-1, expand=False):
             else:
                 meta = type(self._series._meta)([" ".join(["a"] * 2 * n)])
                 meta = meta.str.split(n=n, expand=expand, pat=pat)
+        else:
+            meta = list
```

Comment:
Ah, actually, can you try `meta = (self._series.name, list)` here? Will that work?

Right now we're dropping Series.name, if it has one.

Reply:
`list` didn't work but `object` did :+1: